tests/utils/test_simple_ql.py

```diff
@@ -6,6 +6,7 @@
 
 
 class TestContextBroker(unittest.TestCase):
+
```

Comment:
Test is not from me (only apperead in changelog, because I once added a skip statement)
I would add a comment, but I am honestly a bit unsure what it tests exactly and why it is named as it is